File: python/1742A-Sum.py
n = int(input())

for i in range(1, n+1):
    a, b, c = map(int, input().split())
    m = max(a, b, c)

    if m == a:
        if m == b + c:
            print("YES")
        else:
            print("NO")
    elif m == b:
        if m == a + c:
            print("YES")
        else:
            print("NO")
    elif m == c:
        if m == a + b:
            print("YES")
        else:
            print("NO")

File: C++/1742A - Sum.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, a, b, c, m;
    cin>>n;

    while(n--)
    {
        cin>>a>>b>>c;

        m = max(max(a,b),c);

        if(m==a)
        {
            if(m==b+c)
            {
                cout<<"YES"<<endl;
            }
            else
            {
                cout<<"NO"<<endl;
            }
        }
        else if(m==b)
        {
            if(m==a+c)
            {
                cout<<"YES"<<endl;
            }
            else
            {
                cout<<"NO"<<endl;
            }
        }
        else if(m==c)
        {
            if(m==a+b)
            {
                cout<<"YES"<<endl;
            }
            else
            {
                cout<<"NO"<<endl;
            }
        }
    }

    return 0;
}